docs/usage/configuration-options.md

```diff
@@ -876,7 +876,21 @@ Only the `json` and `yaml` formats are supported.
     }
   ]
 }
-```
+
+```json title="Parsing a TOML file with a custom manager"
+{
+  "customManagers": [
+    {
+      "customType": "jsonata",
+      "fileFormat": "toml",
+      "fileMatch": ["file.toml"],
+      "matchStrings": [
+        "packages.{ \"depName\": package, \"currentValue\": version }"
```

Comment:
Does this work with single strings instead?

Reply:
Sorry, I don't get the question.

Reply:
I meant to write "single quotes" - have since edited it.

e.g. instead of:

```
"packages.{ \"depName\": package, \"currentValue\": version }"
```

Can we do:

```
"packages.{ 'depName': package, 'currentValue': version }"
```

?

Reply:
Yes

Reply:
Updated the docs to use single quotes